src/traits.rs

```diff
@@ -147,6 +160,7 @@ where
         dc: DC,
         rst: RST,
         delay: &mut DELAY,
+        delay_ms: u8,
```

Comment:
1. Would it make sense to make this an option? (longer explanation in the final comment)
```suggestion
        delay_ms: Option<u8>,
```